plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -1214,7 +1218,7 @@ class RocketsPlugin::Impl
             PropertyMap props = object.getPropertyMap();
             if (::from_json(props, request.message))
             {
-                object.updateProperties(props);
+                object.updateProperties(props, false);
```

Comment:
Is it OK to not update the UI if you make changes from a notebook? Or does the rebroadcast at the bottom solve that problem?

Reply:
Try it out :)

Reply:
Yes, the rebroadcast solves that. The callback triggered by markModified() is needed for 'internal' and plugin changes.